Comp2023/src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -372,10 +290,10 @@ public void driveWithPathFollowerInit(PathPlannerTrajectory trajectory, boolean
     m_trajTimer.start( );
   }
 
-  public void driveWithPathFollowerExecute( )
+  public void driveWithPathFollowerExecute(boolean useVision)
   {
     Trajectory.State trajState = m_trajectory.sample(m_trajTimer.get( ));
-    Pose2d currentPose = getPose( );
+    Pose2d currentPose = (useVision) ? m_poseEstimator.getEstimatedPosition( ) : getPose( );
```

Comment:
Not sure why driveWithPathFollower is using a different pose than the robot pose here. It should only follow the path passed in during Init, and it should use the robot pose from getPose.